src/shared/views/ChangeDetection/index.js

```diff
@@ -50,6 +29,7 @@ type Props = {
 export default class ChangeDetectionScreen extends React.Component<Props> {
     randomSeed: number;
 
+    // TODO: this is not used at the moment.
```

Comment:
eventually we could also remove the code below as it is not used anymore

Reply:
Not exactly sure which code you're referring to, but if it's not used, I'd say we remove it now. Or are you planning on bringing it back soon?

Reply:
I think I will not add this back soon. So let's just remove it for now.